### PR TITLE
Ownership single display and edit 

### PR DESCRIPTION
**Pull request by bdc34**
ARXIVD-27

Adds handling of display and edit of a single paper ownership request.

I run this locally against a local db with: `SQLALCHEMY_DATABASE_URI=mysql://bdc34:yourpasswd@localhost/arXiv  FLASK_APP=admin_webapp/app.py  LOCALHOST_DEV=1 FLASK_DEBUG=1 DEBUG=1 flask run`

### CODE REVIEW

**Comment by bdc34** [comment]
Selection_003

**Comment by jimentwood** [comment]
I'm not sure where the right place for this reminder is, but on the Ownership the section below the line:
"Documents below this line have a matching submission e-mail"
and the fuzzy match list that appears...
should be completely removed from the app.

**Review by bmaltzan — APPROVED** [review]

**Comment by bdc34** [comment]
> I'm not sure where the right place for this reminder is, but on the Ownership the section below the line: "Documents below this line have a matching submission e-mail" and the fuzzy match list that appears... should be completely removed from the app.

@jimentwood  I've removed these. I'll make sure to add you as a review on future tapir replacement PRs.

File: admin_webapp/controllers/ownership.py [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    set a row in admin_log with "add-paper-owner-2"
+
+    See tapir/site-src/admin/code/process-ownership-head.php.m4
+
+    Note: This doesn't do "bulk" mode
+    """
+    session = get_db(current_app).session
+    oreq = data['ownership']
+    if request.method == 'POST':
+        admin_id = 1234 #request.auth.user.user_id
```
Comment (DavidLFielding) [inline admin_webapp/controllers/ownership.py:49]: Remember to set this to a legitimate value.

File: admin_webapp/routes/ownership.py [hunk 2]
```diff
@@ -1,39 +1,46 @@
...
     """Pending ownership requests."""
     args = request.args
     per_page = args.get('per_page', default=12, type=int)
     page = args.get('page', default=1, type=int)
-
-    data = _ownership_listing('pending', per_page, page, 0)
-    data['title'] = "Ownership Requests: Pending"
-    return render_template('ownership/requests.html',
+    data = ownership_listing('pending', per_page, page, 0)
+    data['title'] = "Ownership Reqeusts: Pending"
```
Comment (DavidLFielding) [inline admin_webapp/routes/ownership.py:42]: Requests instead of Reqeusts.

**Review by DavidLFielding — COMMENTED** [review]
Looks good!

I tested the branch on my laptop and the ownership requests worked for me. 

I have a few minor ui comments:

ownership detail page:
    - abs link is missing identifier
    - a rejected ownership request, while displaying in rejected list, will indicate "User already owns this paper!"
    - "User already owns this paper!" - if true, should not be displayed on ownership request detail page.
    - In Tapir, when there is a single pending ownership request, clicking the 'pending' list goes directly to the 'detail page' for the single ownership request.
    
lists:
    - Existing system does not track date of admin action. Accepted/rejected lists use the original request date for sorting. This mean an ownership request I accept/reject won't appear in the recent list of accepted/rejected requests if the original request is older than seven days. Should we keep track of the date of actions? I'm not sure whether this information is retrievable from a log or somewhere else.
    - The prod version has "Viewing 1-15 of 45, [[next]" for navigation. I like the 'next' button. Maybe we can improve the navigation links.
    - Some ownership requests have dozens of authors. A wish list item might be to highlight current author's name in the author list. This is not trivial since I have seen ownership requests where the author name varies across the papers being sought.

**Pull request merged**